packs/bitbucket/rules/post_receive_webhook.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: "post_receive_webhook"
+pack: "bitbucket"
+description: "Trigger ST2 pack deploy for commit on Git branch"
+enabled: true
```

Comment:
I think having a dependency on default packs (chatops, st2, packs) is fine. +1 on merging.